build-tools/xaprepare/xaprepare/Steps/Step_PrepareExternalGitDependencies.cs

```diff
@@ -27,6 +27,10 @@ protected override async Task<bool> Execute (Context context)
 				string destDir = Path.Combine (Configurables.Paths.ExternalGitDepsDestDir, egd.Name);
 				if (!Directory.Exists (destDir)) {
 					var egdUrl = await GetGitHubURL (egd, git);
+					if (egd.Owner == "DevDiv") {
+						egdUrl = GetDevDivUrl (egd);
+						Log.WarningLine ($"You may be prompted to enter DevDiv credentials to clone {egd.Name}. Please navigate to {egdUrl} and click the 'Generate Git Credentials' button to get credentials.");
+					}
```

Comment:
I wonder if there is a behavior difference between macOS and windows for this, I seem to always have to copy in a token on the git cli on macOS. I can add a comment here though to mention that this may be helpful.

Reply:
Updated this warning